packages/flex-plugin-e2e-tests/src/tests/step001.ts

```diff
@@ -3,14 +3,23 @@ import { TestSuite, TestParams } from '../core';
 import { spawn, logResult } from '../utils';
 
 // Install Twilio CLI and Plugins CLI
-const testSuite: TestSuite = async ({ environment, scenario }: TestParams): Promise<void> => {
-  const twilioCliResult = await spawn('npm', ['install', `--prefix=${environment.homeDir}`, '-g', 'twilio-cli']);
+const testSuite: TestSuite = async ({ npm, environment, scenario }: TestParams): Promise<void> => {
+  const cliInstallArgs = ['install', '-g', 'twilio-cli'];
+
+  if (!npm.ignorePrefix) {
```

Comment:
When would we set `ignorePrefix` to true? What's the usecase?

Reply:
Few reasons:
- We want that to be `true` when running on Circle CI
- Windows doesn't play nice with `--prefix`

Additional benefit, global install makes much more sense to me as a use case.